package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperproof/hypersync-sdk",
-  "version": "0.9.3",
+  "version": "0.10.0",
```

Comment:
Bumping minor version because we are adding connection health + error info functionality in a backwards compatible way.